airflow/config_templates/config.yml

```diff
@@ -229,6 +229,21 @@
       type: string
       example: ~
       default: "30"
+    - name: dagbag_import_error_tracebacks
+      description: |
+        Should a traceback be shown in the UI for dagbag import errors,
+        instead of just the exception message
+      version_added: 2.0.0
+      type: boolean
+      example: ~
+      default: "False"
```

Comment:
Default of True is probably okay -- I think this is a long term complaint and I can see very few reasons to have it off (by default)

Reply:
Works for me. Running into some test failures by changing the default, but I'll get them worked out.